binscripts/rvm-installer

```diff
@@ -410,21 +410,14 @@ verify_package_pgp()
   else
     typeset _ret=$?
     log "\
-Warning, RVM 1.26.0 introduces signed releases and automated check of signatures when GPG software found. \
-Assuming you trust Michal Papis import the mpapis public key (downloading the signatures).
+GPG signature verification failed for '$1' - '$3'! Try to install GPG v2 and then fetch our public keys:
 
-GPG signature verification failed for '$1' - '$3'! Try to install GPG v2 and then fetch the public key:
+    ${SUDO_USER:+sudo }${rvm_gpg_command##*/} --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
 
-    ${SUDO_USER:+sudo }gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
-
-or if it fails:
+or when it fails, download and import manually:
```

Comment:
I think it's slightly better to say "or if it fails" here.

Reply:
Sure, fixed!